guides/app-guides/univ3-lp.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Details on Angle UniV3 incentive mechanism for LPs
+---
+
+# UniV3 LP incentives in Angle
+
+Angle implemented a specific mechanism to reward UniV3 liquidity providers according to their added value to the protocol. Basically, it allows to reward LPs more granularily according to the efficiency of the liquidity they provide. In turn, they have the opportunity to provide liquidity in more complex ways that benefit from all Uniswap V3 possibilities.
+
+You can find more details on the mechanism in [the original announcement](https://blog.angle.money/a-new-incentivization-mechanism-for-uniswap-v3-liquidity-8ce32fa611b1). Here, we will detail how the incentives are split and how you can provide liquidity on pools to earn ANGLE rewards.
+
+## Incentives distribution
+
+### General
+
+Every week, the distribution of incentives going to each pool is voted by veANGLE holders [here](https://app.angle.money/#/gauge).
+
+### Uniswap V3 pools
+
+Then, for each Uniswap V3 pools, a specific formula is applied to each swap to see who was providing liquidity and how much. Then, at the end of the week, the numbers are computed by an [off-chain script](https://gist.github.com/Picodes/0b738ec92f7bd72ec6e77ffdf5d1c5e2) to form a specific distribution of rewards within each pool.
```

Comment:
It's not clear from this what's the setup: we need a more global sentence like: Reward voted on the gauges are allocated ...

And then explain the setup (we can copy paste the text from the article)